cargo/workspace.bzl

```diff
@@ -0,0 +1,251 @@
+DEFAULT_REPOSITORY_TEMPLATE = "https://crates.io/api/v1/crates/{crate}/{version}/download"
```

Comment:
Can this, `repositories_bin.bzl` and `repositories.bzl` go in `cargo/cargo_universe`? I'd also imagined the resolver to be in a sub-directory there
```
.
└── cargo_universe
    ├── repositories.bzl
    ├── resolver
    │   ├── Cargo.lock
    │   ├── Cargo.toml
    │   ├── src
    │   └── test
    └── workspace.bzl
```
something to this effect.

Reply:
Also, I feel this shouldn't live in `cargo` anymore as it's not tied directly to cargo (at least on the user's end). Should be it's own top level package.